chart/files/pod-template-file.kubernetes-helm-yaml

```diff
@@ -56,6 +56,7 @@ spec:
       image: {{ template "pod_template_image" . }}
       imagePullPolicy: {{ .Values.images.airflow.pullPolicy }}
       name: base
+      securityContext: {{- .Values.podTemplateSecurity.containerSecurityContext | default (.Values.podSecurity.containerSecurityContext) | toYaml | nindent 4 }}
```

Comment:
This should use `workers.containerSecurityContext` instead of having a separate param.